ppr-ui/src/store/getters/state-getters.ts

```diff
@@ -725,18 +725,13 @@ export const getMhrTransferHomeOwners = (state: StateIF): MhrRegistrationHomeOwn
   return owners
 }
 
-export const getMhrTransferCurrentHomeOwners = (state: StateIF): MhrRegistrationHomeOwnerGroupIF[] => {
-  const owners = []
+export const getMhrTransferCurrentHomeOwners = (state: StateIF): any => {
+  const ownerGroups = []
+
   state.stateModel.mhrTransfer.currentOwnerGroups.forEach(group => {
```

Comment:
If we are not doing anything with the groups then maybe we don't need to iterate and simply return it?

Reply:
Was bumping into an issue here. It wouldn't resolve without an iteration over this. Seemed like a Framework issue, possible a symptom of our in between Vue state. Not sure why but this works.